rabbit-escape-render/src/rabbitescape/render/Polygon.java

```diff
@@ -0,0 +1,45 @@
+package rabbitescape.render;
+
+import java.util.Vector;
+
+import rabbitescape.engine.util.Util;
+
+public class Polygon
+{
+    public Vector<Integer> x, y;
```

Comment:
I used 2 arrays because 2 are needed in the end for swing's fillPolygon. However, we have to loop through to scale and offset the coordinates anyway, so we may as well use one list. I agree one list should be neater.